Algorithms/Sequences/A000045_FibonacciSequence.cs

```diff
@@ -14,7 +14,7 @@ namespace Algorithms.Sequences
     ///     OEIS: https://oeis.org/A000045.
     /// </para>
     /// </summary>
-    public class FibonacciSequence : ISequence
+    public class A000045_FibonacciSequence : ISequence
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename class 'A000045_FibonacciSequence' to match pascal case naming rules, consider using 'A000045FibonacciSequence'.](https://app.codacy.com/app/siriak/C-Sharp/pullRequest?prid=4650688)